### PR TITLE
feat(tokens): Add ability to reset accounts tokens

### PR DESCRIPTION
**Pull request by vbudhram**
From secondary emails RRA, we need the ability to reset just an accounts tokens. This adds an endpoint that removes `passwordChangeTokens`, `passwordForgotTokens` and `accountResetTokens`.

Ref: https://github.com/mozilla/fxa-auth-server/issues/1948

Lets merge https://github.com/mozilla/fxa-auth-db-mysql/pull/248 before this one, @philbooth my turn to resolve conflicts 😄

### CODE REVIEW

File: lib/db/mysql.js [hunk 1]
```diff
@@ -1189,6 +1189,14 @@ module.exports = function (log, error) {
     return this.readFirstResult(CONSUME_SIGNIN_CODE, [ createHash(code), newerThan ])
   }
 
+  const ACCOUNT_RESET_TOKENS = 'CALL resetAccountTokens_1(?)'
```
Comment (philbooth) [inline lib/db/mysql.js]: Not sure how important they are but should this have one of the boilerplate `// Delete : ... Where : ...` comments like the other functions in here?

**Comment by philbooth** [comment]
> Lets merge #248 before this one...

Done, thanks!

This looks good to me fwiw, r+. 👍

**Comment by vbudhram** [comment]
Updated per phil and added docs, @mozilla/fxa-devs final r?

**Comment by philbooth** [comment]
> final r?

Beautiful! r+

**Pull request merged**